easy_my_coop_loan/models/loan.py

```diff
@@ -14,14 +14,18 @@ class LoanIssue(models.Model):
     _description = "Loan Issue"
 
     @api.multi
-    def _compute_subscribed_amount(self):
+    def _compute_amounts(self):
         for issue in self:
             susbscribed_amount = 0.0
+            paid_amount = 0.0
             for line in issue.loan_issue_lines.filtered(
                 lambda record: record.state != "cancelled"
             ):
                 susbscribed_amount += line.amount
+                if line.state in "paid":
+                    paid_amount += line.amount
             issue.subscribed_amount = susbscribed_amount
+            issue.paid_amount = paid_amount
```

Comment:
Do we like stateless declarative sentences? YES we do 🦄 
```py
subscription_lines = issue.loan_issue_lines.filtered(
    lambda line: line.state != "cancelled"
)
issue.subscribed_amount = sum(subscription_lines.mapped("amount"))
paid_lines = issue.loan_issue_lines.filtered(
    lambda line: line.state == "paid"
)
issue.paid_amount = sum(paid_lines.mapped("amount"))
```

Reply:
"This is the Way"